README.md

```diff
@@ -1,38 +1,31 @@
-## gifs\_go
+# gifs go client [![GoDoc](https://godoc.org/github.com/gifs/gifs-go?status.svg)](https://godoc.org/github.com/gifs/gifs-go)
 
-[![GoDoc](https://godoc.org/github.com/gifs/gifs_go?status.svg)](https://godoc.org/github.com/gifs/gifs_go)
+Golang package for interacting with the [gifs.com API](https://github.com/gifs/api) to transcode and import media into `.mp4`, `.jpg`, and `.gif` formats. Import all your videos by uploading or passing in any source. It is super easy to import media and integrate in your code.
 
-Go package for interacting with the gifs.com API for transcoding and importing
-media to make at least .MP4, .JPG, .GIF outputs.
+You can see full examples and all the convenience methods in these files:
 
-Import all your videos by uploading and by passing in any source.
+- [`examples/import.go`](https://github.com/gifs/gifs_go/blob/master/examples/import.go)
```

Comment:
Oops, since this path changed from the last commit `/examples/import.go` -> `/example_test.go`.